docs/src/Groups/grouplib.md

```diff
@@ -17,19 +17,39 @@ Pages = ["grouplib.md"]
 
 ## Transitive permutation groups of small degree
 
+TODO: explain about the scope of this.
+
+TODO: give proper attribution to the transgrp package (in particular, cite it)
+
+The arrangement and the names of the groups of degree up to 15 is the same as given in
+[CHM98](@cite). With the exception of the symmetric and alternating group (which are represented
+as `SymmetricGroup` and `AlternatingGroup`) the generators for these groups also conform to this paper with
```

Comment:
`SymmetricGroup` and `AlternatingGroup` are the GAP function names, should we write `symmetric_group` and `alternating_group` here?